internal/cli/client/client.go

```diff
@@ -21,7 +21,8 @@ type Client interface {
 }
 
 type GrpcClient struct {
-	client                protobuf.ClientCPServicesClient
```

Comment:
How about replacing `ClientCPServices` to `ClientServices` and also for `ExecutorCPServices` to `ExecutorServices` everywhere in the project ? here cp is common for both and we know that each part is connected to cp only. I guess we also discussed it in the meeting.